jquery.dynatable.js

```diff
@@ -315,6 +315,11 @@
 
       td += '"';
     }
+    
+    // allow cells to be editable
+    if (column.contentEditable) {
+      td += ' contenteditable="true"';
```

Comment:
This should be `contentEditable="true"` for the `td` attribute.

Also, is setting the column header to contentEditable the best way to allow this feature on columns? Wouldn't the more common use-case be that you want only the cells to be contentEditable and not the header row?